Patches/Moyo from the depth/ThingDefs_Misc/Moyo_Royal_Apparel.xml

```diff
@@ -67,6 +67,32 @@
 				</value>
 			</li>
 			
+			
+			<!--Exotic Dress-->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Moyo_RoyalBlackDress"]/statBases</xpath>
+				<value>
+					<Bulk>7.5</Bulk>
+					<WornBulk>1</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Moyo_RoyalBlackDress"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>5</ArmorRating_Sharp>

```

Comment:
It seems odd to me that what is visibly a fairly skimpy dress, unless we're assuming it's made of something like hyperweave (1mm, 1.5 MPa)? If so, I might suggest using more synthread-like stats (0.2mm, 0.05 MPa), unless this apparel is also quite good armor in the base mod, too?

Reply:
Has the exact same armor as the other noble apparel (mechanically, it blocks middle which prevents it from being used with most armor options anyway), and also does actually use hyperweave in the recipe.